src/Plugins/Nop.Plugin.ExternalAuth.Facebook/Infrastructure/RouteProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Routing;
+using Nop.Web.Framework.Mvc.Routing;
+
+namespace Nop.Plugin.ExternalAuth.Facebook.Infrastructure
+{
+    /// <summary>
+    /// Represents plugin route provider
+    /// </summary>
+    public class RouteProvider : IRouteProvider
+    {
+        /// <summary>
+        /// Register routes
+        /// </summary>
+        /// <param name="endpointRouteBuilder">Route builder</param>
+        public void RegisterRoutes(IEndpointRouteBuilder endpointRouteBuilder)
+        {
+            endpointRouteBuilder.MapControllerRoute(FacebookAuthenticationDefaults.DataDeletionCallbackRoute, $"DataDeletionCallback/",
```

Comment:
Let's add `facebook` in the route template (e.g. `facebook/data-deletion-callback/`)